kernel/arch/dreamcast/include/arch/args.h

```diff
@@ -14,7 +14,7 @@
     \author Paul Cercueil
 */
 
-#if __SH4_SINGLE_ONLY__
+#if defined(__SH4_SINGLE_ONLY__) && __SH4_SINGLE_ONLY__
```

Comment:
Is there any reason we have to do both of these rather than a typical `#ifdef`?

```suggestion
#ifdef __SH4_SINGLE_ONLY__
```

Reply:
None